ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/_repo-layout/create-run/-create-run-utils.ts

```diff
@@ -401,8 +415,10 @@ export function defaultValues(
               ortRun.jobConfigs.reporter !== undefined &&
               ortRun.jobConfigs.reporter !== null,
             formats:
-              ortRun.jobConfigs.reporter?.formats ||
-              baseDefaults.jobConfigs.reporter.formats,
+              // Convert this to correct plugin configuration
```

Comment:
This should explain a bit more *why* this is needed. Keyword: Retriggering of old runs with "wrong" data. And remember to use a dot at the end of the sentence 😉